packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3210,21 +3225,23 @@ export class ContainerRuntime
 		readAndParseBlob: ReadAndParseBlob,
 		versionId: string | null,
 	): Promise<{ snapshotTree: ISnapshotTree; versionId: string; latestSnapshotRefSeq: number }> {
-		const recoveryMethod = this.mc.config.getString(
-			"Fluid.ContainerRuntime.Test.SummarizationRecoveryMethod",
-		);
-		if (recoveryMethod === "restart") {
+		if (this.abnormalAckRecoveryMethod === "restart") {
```

Comment:
If we want to go ON by default, this FG should be reveresed

Reply:
This will need to be a separate PR. I think there will be some failing e2e tests that I will need to fix/adjust.